prelims/handler.py

```diff
@@ -14,10 +14,12 @@ def __init__(self, path_dir, ignore_files=None, encoding='utf-8'):
         self.processors = []
         self.encoding = encoding
         self.ignore_files = [] if ignore_files is None else ignore_files
+        self.lower_path = lower_path
 
     def register_processor(self, processor):
         """Add a front matter processor to the queue.
         """
+        processor.lower_path = self.lower_path
```

Comment:
`processor` can be anything other than `Recommender`, and it's not always true the processor has an attribute `lower_path`. I feel you don't need to update `StaticSitePostsHandler` for the time being; as long as `Recommender` is instantiated with its own `lower_path` attribute, it should do the job.

Apologies my suggestion "configureable at StaticSitePostsHandler level" was misleading; I was thinking something more generic and widely-applicable implementation (in the context of #6), but you don't need to worry about it.

Reply:
Ah, okay 011e793